### PR TITLE
Perl pdl

### PR DESCRIPTION
**Pull request by wlmb**
## Description
Perl solution using the Perl Data Language

## Contributing requirements

* [x] I read the contribution guidelines in CONTRIBUTING.md.
* [x] I placed my solution in the correct solution folder.
* [x] I added a README.md with the right badge(s).
* [x] I added a Dockerfile that builds and runs my solution.
* [x] I selected `drag-race` as the target branch.
* [x] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

**Comment by wlmb** [comment]
I just wonder about 'faithfullness'. My solution does use external packages: PDL and PDL::NiceSlice. However, they are general purpose extensions to the Perl language in order to do number crunching efficiently, and are not packages oriented to any particular problem, such as the prime sieve.

**Comment by rbergen** [comment]
Thanks for contributing this. After looking at it and PDL in general, I think that in the context of this project we should probably consider PDL a language in itself. That means the solution should be put in a directory called PrimePDL/solution_1. For one, it will solve the issue of faithfulness in relation to the inclusion of PDL and PDL::NiceSlice, as those are more or less an integrated part of any PDL program.

**Comment by wlmb** [comment]
Done!
Regards.

File: PrimePDL/solution_1/primes.pl [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    for(my $factor=3; $factor<=$q; $factor+=2) {
+	$bits($factor*$factor:-1:2*$factor).=1 unless $bits(($factor));
+    }
+    $self->{ran}=1;
+}
+
+sub print_results {
+    my ( $self, $show_primes, $show_stats, $duration, $passes ) = @_;
+    say $self->get_primes if $show_primes;
+    printf "Luis Mochán (wlmb) Perl/PDL;%d;%f;%d;algorithm=base,faithful=yes\n",
```
Comment (rbergen) [inline PrimePDL/solution_1/primes.pl]: Please make sure your label (the first field in your output) doesn't contain any spaces. I think replacing them with underscores (_) is an easy fix in this case, but do feel free to use a different approach if you're so inclined.

Comment (wlmb) [reply PrimePDL/solution_1/primes.pl]: Done!

**Review by rbergen — APPROVED** [review]
LGTM

**Review by marghidanu — APPROVED** [review]

**Pull request merged**